### ISSUE TITLE
Ron does not properly round trip through Value

### ISSUE THREAD

**Issue by jrmuizel** [ron-rs/ron#121]
```rust
#[macro_use]
extern crate serde;

extern crate ron;
use ron::de::*;
use ron::ser;
use ron::value::Value;

#[derive(Serialize)]
struct Scene(Option<(u32, u32)>);

#[derive(Serialize)]
struct Scene2 {
    foo: Option<(u32, u32)>,
}

fn main() {
    {
        let s = ser::to_string(&Scene2 { foo: Some((1, 1)) }).unwrap();
        println!("{}", s);
        let scene: Value = from_str(&s).unwrap();
        println!("{:?}", scene);
    }
    {
        let s = ser::to_string(&Scene( Some((1, 1)) )).unwrap();
        println!("{}", s);
        let scene: Value = from_str(&s).unwrap();
        println!("{:?}", scene);
    }
}
```

The first example gives:
```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Parser(ExpectedIdentifier, Position { col: 12, line: 1 })', libcore/result.rs:945:5
```

The second gives:
```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Parser(ExpectedMapColon, Position { col: 6, line: 1 })', libcore/result.rs:945:5
```

**Comment by jrmuizel** [comment]
It looks even plain tuples don't work:

```rust
    {
        let s = ser::to_string(&(1, 1) ).unwrap();
        println!("{}", s);
        let scene: Value = from_str(&s).unwrap();
        println!("{:?}", scene);
    }
```

gives:
```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Parser(ExpectedIdentifier, Position { col: 2, line: 1 })', libcore/result.rs:945:5
```

**Comment by torkleyy** [comment]
Sounds really strange, I'm gonna look at it tomorrow. Are you sure you're
using a recent version?

**Comment by jrmuizel** [comment]
Yeah this is using 0.3.0. How is it supposed to work? deserialize_any doesn't call deserialize_tuple.

**Comment by torkleyy** [comment]
Huh, it seems there are a couple of things not implemented for values. I'm surprised this wasn't noticed earlier.

**Comment by torkleyy** [comment]
@jrmuizel Sorry for the inconvenience, I've opened a PR to fix this: #124.

**Comment by torkleyy** [comment]
I've just merged it, please let me know if there's anything else that's not working properly.
